### ISSUE TITLE
Windows support?

### ISSUE THREAD

**Issue by mooop12** [robertknight/Qt-Inspector#10]
As in the title. I have a Windows-only application I would like to use this tool on. Any chances of adding support for it?

**Comment by robertknight** [comment]
Windows support would be welcome but I'm afraid I don't have the time to work on it myself. For anyone who is interested in having a look, you'll need to create an implementation of the [Injector](https://github.com/robertknight/Qt-Inspector/blob/master/Injector.h) class which is able to load the helper library into the target process. Have a look at the `PreloadInjector` or `GdbLibraryInjector` to see how the implementations for Mac/Linux work.

**Comment by yanzixiang** [comment]
> As in the title. I have a Windows-only application I would like to use this tool on. Any chances of adding support for it?

@mooop12 
I also want to use this tool on windows , have you make it ?

**Comment by tansautn** [comment]
> Windows support would be welcome but I'm afraid I don't have the time to work on it myself. For anyone who is interested in having a look, you'll need to create an implementation of the [Injector](https://github.com/robertknight/Qt-Inspector/blob/master/Injector.h) class which is able to load the helper library into the target process. Have a look at the `PreloadInjector` or `GdbLibraryInjector` to see how the implementations for Mac/Linux work.

So, it doesn't just build the process for Windows, right? I thought QT used the same API for different platforms?

Or do you mean that the process initialization on the Windows platform is different from UNIX?
Making the injection need to be implemented separately for Windows?

And is it work on Linux under WSL ?

**Comment by robertknight** [comment]
> Or do you mean that the process initialization on the Windows platform is different from UNIX?
> Making the injection need to be implemented separately for Windows?

Yes, this is exactly the issue. Most of the code in this tool uses cross-platform Qt APIs, but the code for injecting it into the process is operating system specific.

> And is it work on Linux under WSL ?

It probably does work under WSL, but I haven't tried it.
